src/framework/utils/__tests__/SessionUtils.test.ts

```diff
@@ -5,7 +5,18 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-import { getLoginPageURL } from '../SessionUtils'
+import { renderHook } from '@testing-library/react-hooks'
+import { act } from 'react-dom/test-utils'
```

Comment:
You should grab `act` from `@testing-library/react-hooks` instead
```suggestion
import { act, renderHook } from '@testing-library/react-hooks'
```